roles/setup_base_config/templates/sourceme_common.sh.j2

```diff
@@ -7,6 +7,9 @@ export CHARON_BASE_URL={{ charon_base_url }}
 # Hopefully possible to remove dependency in a later version
 # module load bioinfo-tools piper/{{ piper_module_version }}
 
+# Load latest version of dds-cli
+module load bioinfo-tools dds-cli
```

Comment:
I wonder if we should source a specific version (the latest). 

Pros: It will be documented which version of DDS that is loaded in this version of the environment
Downside: We will have to maintain this line

I think it's OK to skip it, we can leave it to the user to document which version they have used. Just worth considering. What do you think?